pandas/core/resample.py

```diff
@@ -426,7 +430,11 @@ def _groupby_and_aggregate(self, how, *args, **kwargs):
         """
         grouper = self.grouper
 
-        obj = self._selected_obj
+        if self._selected_obj.ndim == 1:
```

Comment:
Unfortunately no; `obj_with_exclusions` will always be a DataFrame even when the input is a Series. On the other hand, `_selected_obj` will be a Series or DataFrame depending on the input, and will only exclude columns when using the `_group_selection_context` method. However we can't use the context because it's only on GroupBy currently and Resampler is a subclass of BaseGroupBy.

I do plan on working to clean this up with the ultimate goal of removing either `obj_with_exclusions` or `selected_obj` in the future.